### PR TITLE
Ask VA 1357 - Dashboard fixes for personal and business tabs and replies

### PR DESCRIPTION
**Pull request by hemeshvpatel**
## Are you removing, renaming or moving a folder in this PR?
- [x] No, I'm not changing any folders (skip to TeamSites and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

Did you change site-wide styles, platform utilities or other infrastructure?
- [x] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- Fix dashboard to show up for authenticated users
- Update date formatter to not error out for invalid date strings
- Update mock data to have business inquiries
- Update styling for personal and business tabs
- Unused question clean up
- The Ask VA team owns this page and it is part of our POC

## Related issue(s)

- Link to ticket here: https://github.com/department-of-veterans-affairs/ask-va/issues/1357

## Testing done

- Manual testing

## Screenshots

image

## What areas of the site does it impact?

- This only impacts the Ask VA Form

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [x] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: src/applications/ask-va/containers/ResponseInboxPage.jsx [hunk 1]
```diff
@@ -62,48 +56,50 @@ const ResponseInboxPage = () => {
...
-          `${envUrl}${baseURL}/inquiries/${temporaryTestInquiryId}${
-            URL.SEND_REPLY
-          }`,
-        );
-      }
-    },
+  const handleSubmit = () => {
+    if (sendReply.reply) {
+      postApiData(
+        `${envUrl}${baseURL}/inquiries/${inquiryId}${URL.SEND_REPLY}`,
```
Comment (ediiotero) [inline src/applications/ask-va/containers/ResponseInboxPage.jsx:78]: Doesn't the URL object have part of this path already?

File: src/applications/ask-va/constants.js [hunk 2]
```diff
@@ -15,8 +15,9 @@ export const URL = {
   GET_HEALTH_FACILITY: `${baseURL}/health_facilities`,
   GET_SCHOOL: `${baseURL}/education_facilities/`,
   SEND_REPLY: `/reply/new`,
-  GET_INQUIRIES: `${baseURL}/inquiries?user_mock_data=true`,
-  // GET_INQUIRIES: `${baseURL}/inquiries`,
+  GET_INQUIRIES: `${baseURL}/inquiries`,
+  GET_INQUIRY: `${baseURL}/inquiries`,
```
Comment (ediiotero) [inline src/applications/ask-va/constants.js:19]: GET_INQUIRIES and GET_INQUIRY are the same string. Do we need both?

**Review by ediiotero — DISMISSED** [review]
Looks good just had a couple questions

**Review by ediiotero — APPROVED** [review]
looks good

**Review by pjhill — APPROVED** [review]

**Pull request merged**